en/docs/develop/creating-artifacts/creating-endpoint-templates.md

```diff
@@ -1,75 +1,93 @@
-# Creating Endpoint Templates
+# Create an Endpoint Template
 
-Follow the instructions given below to create a new **Endpoint Template** in WSO2 Integration Studio.
+Follow the instructions given below to create a new Endpoint Template in the Micro Integrator for Visual Studio Code extension (MI for VS Code).
 
 ## Instructions
 
-### Creating the Endpoint Template artifact
+### Create the endpoint template artifact
 
-1.  Right-click the [ESB Config project]({{base_path}}/develop/create-integration-project/#esb-config-project) and go to **New → Template** to open the **New Template Artifact** dialog box.
+{!includes/creating-project.md!}
 
-    <img src="{{base_path}}/assets/img/integrate/create_artifacts/new_template/select-template.png">
+3. Go to **Micro Integrator Project Explorer** > **Templates**. 
 
-2.  Select **Create a New Template** and click **Next**.
+    <a href="{{base_path}}/assets/img/develop/create-artifacts/create-endpoint-template/create-new-template.png"><img src="{{base_path}}/assets/img/develop/create-artifacts/create-endpoint-template/create-new-template.png" alt="Create endpoint template" width="30%"></a>
 
-    <img src="{{base_path}}/assets/img/integrate/create_artifacts/new_template/new-template-wizard-1.png" width="500">
+4. Hover over **Templates** and click the **+** icon that appears to open the below **Template Form**.
 
-3.  Enter a unique name for the template and select one of the following **Endpoint Template** types.
+    <a href="{{base_path}}/assets/img/learn/tutorials/add-template.png"><img src="{{base_path}}/assets/img/learn/tutorials/add-template.png" alt="Add-template" width="30%"></a>
 
-    <img src="{{base_path}}/assets/img/integrate/create_artifacts/new_template/new-template-wizard-2-1.png" width="500">
+5. This will open the **Template Form** shown below, from which you can select the **Template Artifact**. Select the endpoint template type.
```

Comment:
```suggestion
5. This will open the **Template Form** shown below, where you can select the **Template Artifact**. Select the endpoint template type.
```